packages/preact/test/index.test.tsx

```diff
@@ -189,6 +189,70 @@ describe("@preact/signals", () => {
 			rerender();
 			expect(spy).to.be.calledOnce;
 		});
+
+		it("should not subscribe to computed signals only created and not used", () => {
```

Comment:
@developit Does this test capture the behavior we've spoken about in some of our meetings re: creating and passing down computed values. Just figured I'd add a test to ensure the core integration libraries have the behavior we expect.